## Leer los datos iniciales
Importamos nuestro excel y damos un vistazo a las hojas que tiene

In [11]:
import pandas as pd

xls = pd.ExcelFile('../data/raw/datos-siniestros.xlsx')

## Exploracion inicial de las hojas del archivo:
En esta sección se analiza el número de filas y columnas de cada hoja del Excel, con el fin de distinguir entre hojas con datos reales y aquellas que funcionan como diccionarios o catálogos de referencia.

In [13]:
resumen = []
for hoja in xls.sheet_names:
    df = pd.read_excel(xls, sheet_name=hoja)
    resumen.append({"Hoja": hoja, "Filas": df.shape[0], "Columnas": df.shape[1]})

pd.DataFrame(resumen)

Hoja  Filas  Columnas
0              Campos     24       519
1  Tabla de Contenido    384         5
2    Base Consolidada    105      1012
3              Hoja 2    246       120
4              Hoja 3    124         3
5     Actores viales      46         8
6           Vehículos     46        49
7       Entorno y vía     46        44
8  Factores de riesgo     66         5

Con esta informacion puedo identificar:

Las hojas con pocas filas pero muchas columnas suelen corresponder a diccionarios de datos, donde cada columna representa una variable y las filas contienen sus atributos descriptivos (nombre, tipo, unidades, etc.)

#### Localidad (Una de las 519 columnas) tiene solo 20 filas las cuales agrupan todas las localidades existentes, pero no es un dato que describa un caso real en especifico

Aquellas hojas que tienen muchas filas pero pocas columnas, suelen ser al reves, aquellas hojas que describen listas de referencia, como tablas de contenido

Nos interesan principalmente las hojas que presentan un número significativo tanto de filas como de columnas, ya que son las que probablemente contienen registros reales.

## Conclusion parcial:
-Las hojas 'Campos', 'Tabla de contenido','Actores viales','Vehiculos','Entorno y via' y 'Factores de riesgo' corresponden a diccionarios o catalogos

-Las hojas 'Hoja 2' y 'Hoja 3' parecen ser hojas ocultas o auxiliares

-La hoja **Base Consolidada** contiene los datos reales de los siniestros y sera la que analicemos de ahora en adelante